de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/core/ui/util/CollaborationUtils.java

```diff
@@ -255,9 +254,9 @@ private static String getShareProjectDescription(
                 Pair<Long, Long> fileCountAndSize;
 
                 if (sarosSession.isCompletelyShared(project)) {
-                    fileCountAndSize = FileUtils
-                        .getFileCountAndSize(Arrays.asList(project.members()),
-                            true, IContainer.FILE);
+                    fileCountAndSize = getFileCountAndSize(
+                        Arrays.asList(project.members()), true,
+                        IContainer.FILE);
 
                     result.append(String
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: %n should be used in place of \n to produce the platform-specific line separator. ([squid:S3457](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3457))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=308&issues=AWcYCgrzkEeD8QRk2pY2&open=AWcYCgrzkEeD8QRk2pY2)

[ISSUE_KEY]: AWcYCgrzkEeD8QRk2pY2